charts/consul/test/unit/server-disruptionbudget.bats

```diff
@@ -97,7 +97,7 @@ load _helpers
       --set 'server.replicas=6' \
       . | tee /dev/stderr |
       yq '.spec.maxUnavailable' | tee /dev/stderr)
-  [ "${actual}" = "2" ]
+  [ "${actual}" = "1" ]
```

Comment:
**suggestion:** This file too

Reply:
Corrected as recommended by reverting back to `release/1.3.x` branch version of affected files.

```shell
$ git checkout 'release/1.3.x' -- charts/consul/templates/server-disruptionbudget.yaml charts/consul/test/unit/server-disruptionbudget.bats charts/consul/template/_helpers.tpl
```
Applied datadog-integration changes back into `_helpers.tpl`

Re-ran entirety of bats tests using `Makefile` - `make bats-tests` (all passed)